Try to clarify interaction of PADDING and ACK (#1923)
quicwg/base-drafts: draft-ietf-quic-transport.md

however acknowledge packets containing only ACK or PADDING frames when sending
ACK frames in response to other packets.

> While PADDING frames are only eventually acknowledged, they are still considered

Reviewer: The word `only` here is awkward to me. I think it would be best removed.

> While PADDING frames are only eventually acknowledged, they are still considered

Reviewer 2: ```suggestion
While PADDING frames may never be acknowledged, they are still considered
```

Author: This is hard.  never acknowledged isn't true either.  The point I'm trying to capture is that a packet containing PADDING doesn't cause the recipient to start their delayed acknowledgment timer.  Keep in mind that this isn't critical language.  Maybe instead.

```suggestion
Packets containing PADDING frames are still considered
```
